src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,175 @@
+
+Learn how to quickly grant and revoke access to an organization's resources via the web user interface, using Kubernetes role based access control (RBAC) mechanisms.
+
+{{% organization_mgmt_mapi_disclaimer %}}
+
+Granting access means that the users (when associating users or groups) or programs (when using service accounts) affected will be able to access resources via the management API, including via the web user interface itself.
+
+The access control UI in the web interface provides quick access to a focused set of RBAC resources, with the goal to make it easy for you to give someone – be it individuals or groups – or service accounts access to resources in an organization's namespace.
+
+## Where to find it {#where-to-find}
+
+1. Log in to the web user interface, using the URL matching your management cluster/installation.
+2. Click **Organizations** in the main menu in the top.
+3. Click the organization you want to manage access for.
+4. Select the **Access control** tab.
+
+## User interface overview {#ui-overview}
+
+![Access control user interface overview](access-control-ui.png)
+
+The access control user interface provides:
+
+- A list of roles to select from, sourced from both the organization's namespace and cluster roles. Each role defines a specific set of permissions for a number of resources. Cluster roles are marked using a 🌐 icon. In the list, it is also indicated how many users and groups are bound to each role in this organization's namespace.
+
+- For the selected role:
+
+    - Basic information regarding whether the role is a cluster role or a namespaced role, and who manages the role according to the `app.kubernetes.io/managed-by` label (defaulting to **You** if the label does not exist).
+
+    - The list of **Subjects** associated with the role. Subjects are individual users or user groups, as defined in the identity provider you use for authentication.
+
+    - The list of **Permissions** granted to any subject associated with the role.
+
+While the **Subjects** tab will be explained in more detail below under [granting access](#granting-access), the information shown under **Permissions** deserves a closer look here.
+
+### Permissions
+
+![Access control user interface overview](permissions.png)
+
+The example above shows permission details for a role that grants:
+
+- all permissions to `apps` and `charts` from the `application.giantswarm.io` API group.
+- only `get`, `list`, and `watch` permission to `appcatalogs` and `appcatalogentries` from the `application.giantswarm.io` API group.
+- only `get`, `list`, and `watch` permission for clusters.
+
+The single letter icons in the **Verbs** column each represent one of the seven standard verbs: `get`, `watch`, `list`, `create`, `update`, `patch`, and `delete`. The last items, showing an ellipsis `⋯`, stands for additional, arbitrary verbs. Hovering over these icons with your pointer will reveal additional information in a tooltip.
```

Comment:
```suggestion
The single letter icons in the **Verbs** column each represent one of the seven standard verbs: `get`, `watch`, `list`, `create`, `update`, `patch`, and `delete`. The last items, showing an ellipsis `⋯`, stands for additional, arbitrary verbs. Hovering over these icons with your pointer will reveal additional information in a tooltip.
```
I'm not sure what arbitrary means here. I'm pretty sure you didn't mean random here. Maybe we can find a less arbitrary word 😉

Reply:
Kubernetes allows to use any string as a "verb" for permissions. The verbs are meant as in `<subject> can to <verb> to <object>`. It is completely up to the user to invent things like `run`, `check` or `banana`.